### DISCUSSION TITLE
Why isn't the plugins folder created?

### DISCUSSION THREAD

**Discussion by dongdong0825** [IzzelAliz/Arclight#939] [General]
I created a server, but the plugins folder is not created. Even if you create it yourself, it cannot be applied. 1.19.2 as well as 1.19.3

**Comment by heipiao233** [comment]
Hello, Forge will rewrite run.bat and run.sh, please remove them and create runarc.bat for Windows or runarc.sh for Linux. Use them to start.

**Reply by DutchBrotherz** [reply]
What if it's on a minecraft hosting platform?
